lib/mix/test/mix/tasks/release_test.exs

```diff
@@ -786,6 +786,60 @@ defmodule Mix.Tasks.ReleaseTest do
     end)
   end
 
+  test "works properly with an absolute symlink to release" do
+    in_fixture("release_test", fn ->
+      Mix.Project.in_project(:release_test, ".", fn _ ->
+        Mix.Task.run("release")
+
+        script_name = "release_test"
+
+        script_name =
+          if match?({:win32, _}, :os.type()), do: "#{script_name}.bat", else: script_name
+
+        File.ln_s!(
+          Path.join("_build/dev/rel/release_test/bin", script_name),
+          script_name
+        )
+
+        try do
+          script = Path.absname(script_name)
+          {hello_world, 0} = System.cmd(script, ["eval", "IO.puts :hello_world"])
+          assert String.trim_trailing(hello_world) == "hello_world"
+        after
+          File.rm!(script_name)
```

Comment:
You don't need to delete it, the whole fixture we created is cleaned up afterwards.